openedx/adg/lms/student/urls.py

```diff
@@ -2,11 +2,10 @@
 Urls for student app
 """
 from django.urls import path
-from django.views.generic import TemplateView
 
-from .views import AccountView
+from .views import AccountView, CoursesView
 
 urlpatterns = [
     path('account/v2/', AccountView.as_view(), name='adg_account'),
-    path('dashboard/v2/', TemplateView.as_view(template_name='student_account/dashboard.html'))
+    path('dashboard/v2/', CoursesView.as_view(), name='adg_dashboard')
```

Comment:
For now it is fine but we should update as it is the dashboard not just the courses.

Reply:
Yes this was just a temporary view, will be updating it under LP-2937